akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -361,26 +352,21 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
       .insert(Array("system", "cluster", "core", "daemon", "crossDcHeartbeatSender"), NotUsed)
       .insert(Array("system", "cluster", "heartbeatReceiver"), NotUsed)
 
-  private def inboundChannel = s"aeron:udp?endpoint=${_bindAddress.address.host.get}:${_bindAddress.address.port.get}"
-  private def outboundChannel(a: Address) = s"aeron:udp?endpoint=${a.host.get}:${a.port.get}"
-
-  private val controlStreamId = 1
-  private val ordinaryStreamId = 2
-  private val largeStreamId = 3
-
-  private val taskRunner = new TaskRunner(system, settings.Advanced.IdleCpuLevel)
+  protected val controlStreamId = 1
```

Comment:
could be constants on the companion object

Reply:
👍

Reply:
Actually not, some fields are set only once in the `Encoder`, those that never change for each message. Adding a comment.

Reply:
Aha gotcha